lib/membrane_h264_plugin/parser/decoder_configuration_record.ex

```diff
@@ -32,27 +76,27 @@ defmodule Membrane.H264.Parser.DecoderConfigurationRecord do
         <<1::8, avc_profile_indication::8, profile_compatibility::8, avc_level::8, 0b111111::6,
           length_size_minus_one::2, 0b111::3, rest::bitstring>>
       ) do
-    {sps, rest} = parse_sps(rest)
-    {pps, _rest} = parse_pps(rest)
+    {spss, rest} = parse_spss(rest)
+    {ppss, _rest} = parse_ppss(rest)
 
     %__MODULE__{
-      sps: sps,
-      pps: pps,
+      spss: spss,
+      ppss: ppss,
       avc_profile_indication: avc_profile_indication,
       profile_compatibility: profile_compatibility,
       avc_level: avc_level,
-      length_size_minus_one: length_size_minus_one
+      nalu_length_size: length_size_minus_one + 1
     }
     |> then(&{:ok, &1})
```

Comment:
I know it's not your part of code, but I think we never return `:error` tuple, so there is no point in returning `{:ok, result}` - we can remove that ;)

Reply:
with pleasure :)